war/src/main/webapp/css/style.css

```diff
@@ -22,6 +22,10 @@
  * THE SOFTWARE.
  */
 
+.item-icon-freestyleproject .icn {
+  background: url('../images/items/freestyle-48.png') center center no-repeat !important;
+}
```

Comment:
:ant: yes, this should not be here. What is it used for exactly? Could it be part of the `TopLevelItemDescriptor.getDescription()`?